bing.com.BingWebMasterDPOOV1.json

```diff
@@ -0,0 +1,22 @@
+{
+  "providerId": "bing.com",
+  "providerName": "Microsoft Bing",
+  "serviceId": "BingWebMasterDPOOV2",
```

Comment:
Is it new template, or a new version of BingWebMasterDPOO ?
The difference is the added syncRedirectDomain, which allows you to use with or without it, so for me it seems backward compatible, so no need to create a new template.
Please also keep the file name in sync with the structure {providerId}.{serviceId}.json